# Get the map as a epthmap_humanoid_msgs/DepthMap
---
DepthMap map
